src/Xlang/UndockedRegFreeWinRT/src/UndockedRegFreeWinRT/UndockedRegFreeWinRT/UndockedRegFreeWinRT.vcxproj

```diff
@@ -251,6 +251,11 @@
       <ModuleDefinitionFile>winrtact.def</ModuleDefinitionFile>
     </Link>
   </ItemDefinitionGroup>
+  <ItemDefinitionGroup Condition="'$(WingetEnableReleaseBuild)'=='true'">
```

Comment:
>WingetEnableReleaseBuild [](http://example.com/codeflow?start=37&length=24)

Release build is confusing, it should be named UseProdWingetServer

Btw, do we have a reason to not use existing UseProdClsid?